python/hopsworks_common/client/base.py

```diff
@@ -168,6 +168,8 @@ def _send_request(
             files=files,
         )
 
+        print("verify:{}".format(self._verify))
```

Comment:
Guessing it is an artefact of debugging? Adding a logger instance for this module and turn it into a debug could be helpful in the future

Reply:
It was, added a logger instead